pom.xml

```diff
@@ -75,11 +75,11 @@
 	</build>
 
 	<distributionManagement>
-		<snapshotRepository>
-			<id>nexus-snapshots</id>
-			<name>Internal Snapshots Releases</name>
-			<url>http://nexus.intra.local.ch:8081/nexus/content/repositories/snapshots/</url>
-		</snapshotRepository>
+		<repository>
+			<id>nexus</id>
+			<name>Internal Releases Repository</name>
+			<url>http://nexus.intra.local.ch:8081/nexus/content/repositories/releases//</url>
```

Comment:
double slash in the end?!

Reply:
![image](https://user-images.githubusercontent.com/783861/30880490-096459ae-a303-11e7-8092-8456ba8c7009.png) ![image](https://user-images.githubusercontent.com/783861/30880490-096459ae-a303-11e7-8092-8456ba8c7009.png)

Reply:
I removed one. But it actually also worked with both. ;-)